cypress/e2e/index/select-bothfiles.cy.js

```diff
@@ -0,0 +1,43 @@
+// Description: User selects data table AND data dictionary
+// Expected results: Data dictionary selection does not enable next page nav, but data table selection does
+
```

Comment:
The testing functionality in this and the following test files in this commit (each `describe` + `it` function definitions) are the first implementation of UI-based testing for the index page before later test organization (`describe`, `context`, and multiple `it` functions per context) was established.